catalog/security/catalog-security-filter/src/test/java/ddf/catalog/security/filter/plugin/test/FilterPluginTest.java

```diff
@@ -1,10 +1,10 @@
 /**
```

Comment:
Comment: Just surprised that this Test class has a relatively large number of "support" methods that are public instead of private.